adding RTMP support

XxMARSxX opened the pull request:
adding RTMP support for using with shortcode

XxMARSxX committed:
Update video-js.php

adding rtmp support

XxMARSxX committed:
Update readme.txt

adding RTMP help

XxMARSxX committed:
Update help.html

add help for RTMP

mmcc commented:
Thanks for the PR! This looks good to me, so I'll pull this in and release next week.

XxMARSxX committed:
Added RTMP in VisualEditor

Added ability to select RTMP stream in the visual editor

XxMARSxX committed:
* Update video.js to 4.11.3
* Update youtube plugin to latest version
* Some fixes in video-js.php

	modified:   video-js.php
	new file:   video-js.php.orig
	modified:   videojs/demo.html
	modified:   videojs/font/vjs.eot
	modified:   videojs/font/vjs.svg
	modified:   videojs/font/vjs.ttf
	modified:   videojs/font/vjs.woff
	new file:   videojs/lang/ar.js
	new file:   videojs/lang/de.js
	new file:   videojs/lang/es.js
	new file:   videojs/lang/fr.js
	new file:   videojs/lang/hu.js
	new file:   videojs/lang/it.js
	new file:   videojs/lang/ja.js
	new file:   videojs/lang/ko.js
	new file:   videojs/lang/nl.js
	new file:   videojs/lang/pt-BR.js
	new file:   videojs/lang/ru.js
	new file:   videojs/lang/uk.js
	new file:   videojs/lang/zh.js
	modified:   videojs/video-js.css
	modified:   videojs/video-js.less
	modified:   videojs/video-js.min.css
	modified:   videojs/video-js.swf
	modified:   videojs/video.dev.js
	modified:   videojs/video.js
	modified:   videojs/vjs.youtube.js

XxMARSxX committed:
Delete video-js.php.orig

XxMARSxX committed:
Update readme.txt

XxMARSxX committed:
Update readme.txt

change my nick to name

XxMARSxX committed:
Update readme.txt

add changelog = 4.11.3 =

XxMARSxX commented:
updated Video.js to latest version and some changes

XxMARSxX committed:
* Fixed YouTube

	modified:   mce-button.js

XxMARSxX committed:
Update readme.txt

XxMARSxX committed:
fixed swf link if self hosted version using
the code:
    if(typeof videojs != "undefined") {
    videojs.options.flash.swf = "'. plugins_url( 'videojs/video-js.swf' , __FILE__ ) .'";
    }
doing nothing and removed from head to the 'data-setup' attribute to automatically set up the video when the page is ready

the code:
    document.createElement("video");document.createElement("audio");document.createElement("track");
not needed because it is called in firs rows of video.js

	modified:   video-js.php
